bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -399,6 +440,10 @@
 		<label>Forecasted Temperature</label>
 		<description>Forecasted outdoor temperature.</description>
 		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
I am wondering here: A forecast is not really a measurement. I'd therefore rather refrain from adding this tag.
We might want to think of something like a "forecast" tag for the tag library instead. Wdyt?

Reply:
You're right, a forecast is not exactly a measurement.
I will add a new Point "Forecast" in the semantics tag.

Reply:
I am going to replace "Measurement" by "Status" for forecast channels.